libdispatch/dfile.c

```diff
@@ -1973,10 +1973,6 @@ check_create_mode(int mode)
     /* mmap is not allowed for netcdf-4 */
     if(mmap && (mode & NC_NETCDF4)) return NC_EINVAL;
 
-    /* Can't use both parallel and diskless|inmemory|mmap. */
```

Comment:
Are we still handling case where nc_create() is called with NC_MPIIO flag? (Which should be an error IIRC.)

Reply:
The current implementation is to ignore NC_MPIIO if used in nc_create and nc_open.
I am OK with making it an error (NC_EINVAL). @DennisHeimbigner @WardF ?

Reply:
I guess if it will not work (i.e. will not give them parallel I/O) it should be considered an error.

Reply:
One question. Once we marked a flag as deprecated, should we still check its use?